src/main/java/com/exasol/validation/GitHubPlatformValidator.java

```diff
@@ -42,7 +42,7 @@ private void validateChangesFile(final ReleaseLetter releaseLetter) {
     protected void validateContainsHeader(final ReleaseLetter changes) {
         final Optional<String> header = changes.getHeader();
         if (header.isEmpty()) {
-            throw new IllegalStateException("The " + changes.getFileName()
+            throw new IllegalStateException("E-VAL-GHPV-1: The " + changes.getFileName()
```

Comment:
```suggestion
            throw new IllegalStateException("E-RR-GHV-1: The " + changes.getFileName()
```
This is something that won't be used outside RR.